DataFormats/Headers/src/DataHeader.cxx

```diff
@@ -27,14 +27,15 @@ const o2::Header::SerializationMethod o2::Header::gSerializationMethodFlatBuf("F
 //__________________________________________________________________________________________________
 //possible data origins
 const o2::Header::DataOrigin o2::Header::gDataOriginAny    ("***");
-const o2::Header::DataOrigin o2::Header::gDataOriginInvalid("   ");
+const o2::Header::DataOrigin o2::Header::gDataOriginInvalid("INV");
```

Comment:
I would  use 'void' instead of 'invalid' in the name, this can probably be changed later, but in any case  simply set it to 0